modules/example/src/smithy4s/example/TestAdt.scala

```diff
@@ -0,0 +1,66 @@
+package smithy4s.example
+
+import smithy4s.ByteArray
+import smithy4s.Hints
+import smithy4s.Schema
+import smithy4s.ShapeId
+import smithy4s.ShapeTag
+import smithy4s.schema.Schema.bytes
+import smithy4s.schema.Schema.int
+import smithy4s.schema.Schema.long
+import smithy4s.schema.Schema.short
+import smithy4s.schema.Schema.string
+import smithy4s.schema.Schema.struct
+import smithy4s.schema.Schema.union
+
+sealed trait TestAdt extends AdtMixinOne with AdtMixinTwo with scala.Product with scala.Serializable {
+  @inline final def widen: TestAdt = this
+}
+object TestAdt extends ShapeTag.Companion[TestAdt] {
+  val id: ShapeId = ShapeId("smithy4s.example", "TestAdt")
+
+  val hints: Hints = Hints.empty
+
+  case class AdtOne(lng: Option[Long] = None, sht: Option[Short] = None, blb: Option[ByteArray] = None, str: Option[String] = None) extends TestAdt with AdtMixinTwo with AdtMixinOne with AdtMixinThree
```

Comment:
Will update so that AdtMixinTwo and AdtMixinOne don't render here. AdtMixinThree still should since it is not a part of the `TestAdt` level.